app/api/v1/conversations.py

```
@@ -0,0 +1,38 @@
+from app.db.models import Conversation
+from app.db.config.database import get_async_session
+
+router = APIRouter()
+
+class ConversationCreate(BaseModel):
+    email: str = None  # Opcional
+
+@router.post("/conversations")
+async def create_conversation(
+    conversation: ConversationCreate,
+    session: AsyncSession = Depends(get_async_session)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)